Games/Warcraft III/Online/script.js

```diff
@@ -0,0 +1,21 @@
+include(["Functions", "QuickScript", "OnlineInstallerScript"]);
+include(["Functions", "Verbs", "corefonts"]);
+include(["Functions", "Verbs", "vcrun2015"]);
+
+new OnlineInstallerScript()
+    .name("Warcraft III")
+    .editor("Blizzard")
+    .applicationHomepage("http://eu.battle.net/en/")
```

Comment:
Please use a homepage for this particular application if possible (e.g. http://eu.blizzard.com/en-gb/games/war3/)